Floofbot/Modules/Administration.cs

```diff
@@ -156,7 +156,14 @@ public async Task warnlog([Summary("user")] string user)
                 .Where(u => u.UserId == badUser.Id && u.GuildId == Context.Guild.Id)
                 .OrderByDescending(x => x.DateAdded).Take(24);
 
+            if (warnings.Count() == 0)
+            {
+                await Context.Channel.SendMessageAsync($"{badUser.Username}#{badUser.Discriminator} is a good noodle. They have no warnings!");
+                return;
+            }
+
             EmbedBuilder builder = new EmbedBuilder();
+            builder.Color = ADMIN_COLOR;
             int warningCount = 0;
```

Comment:
int warningCount should be declared before the builder is, since it's kinda confusing that there's a random int being declared between builder.Color and builder.WithTitle.
I'll let it slide though